grml2usb

```diff
@@ -905,14 +905,20 @@ def unmount(target, unmount_options):
             logging.debug("unregister_mountpoint(%s)", target)
             unregister_mountpoint(target)
 
+def extract_device_name(device):
+    match = re.match(r"/dev/([a-z0-9]*[0-9]*)$", device)
```

Comment:
Please do not change the functionality when refactoring code. (see: https://github.com/grml/grml2usb/pull/41/commits/fd29ffed93f33c8cccb0ba8ca68737156c4a5fb0#diff-2f27a82c8acea6b2b838b2960d531000R918)